site/content/docs/Advanced/Ransomware Protection/_index.md

```diff
@@ -54,7 +54,8 @@ Some cloud storage solutions provide the ability to generate restricted access k
                     ]
                 }
             ]
-        }```
+        }
+        ```
```

Comment:
i think indentatiion is off here...

Reply:
Hey @jkowalski, 

indentation is fixed. Thanks for the feedback.